marklogic-data-hub-central/ui/src/pages/Browse.test.tsx

```diff
@@ -5,12 +5,36 @@ import {MemoryRouter} from "react-router-dom";
 import Browse from "./Browse";
 import {SearchContext} from "../util/search-context";
 import userEvent from "@testing-library/user-event";
+import axiosMock from "axios";
+import {act} from "react-dom/test-utils";
 
 jest.mock("axios");
 jest.setTimeout(30000);
 
-
 describe("Explorer Browse page tests ", () => {
+
+  const modelsResponse = [
+    {
+      "info": {
+        "title": "Customer",
+        "version": "0.0.1",
+        "baseUri": "http://example.com/",
+        "description": "An Customer entity"
+      },
+      "definitions": {
+        "Customer": {
+          "properties": {
+            "FirstName": {
+              "datatype": "string",
+              "collation": "http://marklogic.com/collation/codepoint"
+            }
+          }
+        }
+      }
+
+    }
+  ];
+  const hubConfigResponse = {"modeling": {"entities": {"Customer": {"graphX": 16, "graphY": 54, "color": "#e3deeb", "icon": "FaOdnoklassniki"}}}};
```

Comment:
This also can be moved under `src/assets`, like mentioned in the previous comment.